docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -633,6 +736,22 @@ These rules apply to ``filter_environment`` and ``mask_environment``.
 
     ``mask`` can contain conditional expressions such as ``IF`` or ``CASE``, which achieves conditional masking.
 
+Function rules
+^^^^^^^^^^^^^^
+
+Each function rule is composed of the following fields:
+
+* ``user`` (optional): regex to match against user name. Defaults to ``.*``.
+* ``role`` (optional): regex to match against role names. Defaults to ``.*``.
+* ``group`` (optional): regex to match against group names. Defaults to ``.*``.
+* ``schema`` (optional): regex to match against schema name. Defaults to ``.*``.
+* ``function`` (optional): regex to match against function names. Defaults to
+  ``.*``.
+* ``privileges`` (required): zero or more of ``EXECUTE``, ``GRANT_EXECUTE``.
+* ``function_kinds`` (required): one or more of ``AGGREGATE``, ``SCALAR``,
+  ``TABLES``, ``WINDOW``. When user define a rule for ``AGGREGATE``, ``SCALAR``
+  or ``WINDOW`` function, ``catalog`` and ``schema`` fields are disallowed.
```

Comment:
This appears to duplicate lines 358 to 371. Is this intentional @huberty89 ? If so, all suggestions and comments there apply here also.

Reply:
Everything here is the same expect `role` and `catalog` - I forgot to remove `role` entry before.